apps/examples/performance/tls_seclink/tls_client.c

```diff
@@ -0,0 +1,370 @@
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+/****************************************************************************
+ *
+ * Copyright 2020 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <tinyara/config.h>
+#include <stdio.h>
+
+#include "tls_common.h"
+
+static unsigned char rootca[] = ROOT_CA;
+static int rootca_len = sizeof(rootca);
+
+static const unsigned char TLS_DATA[] = "This is from CLIENT";
+
+#define TEST_CASE_COUNT 7
+static unsigned char test_data[4100] = { 0, };
+static int test_size[TEST_CASE_COUNT] = { 64, 128, 256, 512, 1024, 2048, 4096 };
+static uint32_t test_log[2][TEST_CASE_COUNT] = { 0, };
+static uint32_t log_diff = 0;
+static int log_count[2] = { 0, };
+
+#define ENCRYPT 0
+#define DECRYPT 1
+
+#if defined(CONFIG_TLS_SECLINK_USE_FRT_TIMER)
+static struct timer_status_s before;
+static struct timer_status_s after;
+int frt_fd;
+
+#define TLS_START_TIMER(x) \
+	frt_fd = open(CONFIG_TLS_SECLINK_FRT_TIMER_PATH, O_RDONLY); \
+	ioctl(frt_fd, TCIOC_SETMODE, MODE_FREERUN); \
+	ioctl(frt_fd, TCIOC_START, 0); \
+	ioctl(frt_fd, TCIOC_GETSTATUS, (unsigned long)(uintptr_t)&before);
+
+#define TLS_STOP_TIMER(x) \
+	ioctl(frt_fd, TCIOC_GETSTATUS, (unsigned long)(uintptr_t)&after); \
+	ioctl(frt_fd, TCIOC_STOP, 0); \
+	close(frt_fd); \
+	do { \
+		log_diff = (unsigned long)after.timeleft - (unsigned long)before.timeleft; \
+		test_log[x][log_count[x]++] = log_diff; \
+	} while(0);
+#else
+static struct timespec before;
+static struct timespec after;
+
+#define TLS_START_TIMER(x) clock_gettime(CLOCK_REALTIME, &before);
+
+#define TLS_STOP_TIMER(x) \
+	clock_gettime(CLOCK_REALTIME, &after); \
```

Comment:
For using `clock_gettime()`, please include `time.h`.

Reply:
@jeongchanKim 'time.h' is included in 'tls_common.h'